src/bandersnatch_filter_plugins/latest_name.py

```diff
@@ -32,6 +34,17 @@ def initialize_plugin(self) -> None:
             return
         if self.keep > 0:
             logger.info(f"Initialized latest releases plugin with keep={self.keep}")
+        try:
+            sort_by = self.configuration["latest_release"]["sort_by"]
+            if sort_by in ["time", "version"]:
+                self.sort_by = sort_by
+            logger.info(
+                f"Initialized latest releases plugin with sort_by={self.sort_by}"
+            )
+        except KeyError:
+            return
+        except ValueError:
+            return
```

Comment:
Lets log.error's here ... Unless something upstream does ...

What can throw a ValueError here?

Reply:
this is just copy paste from above (self.keep = int(self.configuration["latest_release"]["keep"])). I did not really think about possible errors, since `keep` is handled the same way...

Reply:
It's an int, so if a str that is not a number is set the ValueError can occur ... Blindly copy pasting without thinking about it should be done sparingly ...

Reply:
Oups, missed that...
What about the KeyError? As far as I understand, nothing should be looged if some optional setting is not in the config file.
I would remove the ValueError and hope that the PR can then be merged:-)